### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by Santosh1176**
Added my profile to /public/data 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1276"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]
The icons that you've listed in your milestones are not currently available in the project so they will default to a globe for the time being. Please feel free to raise separate issues for the icons you wish to be added and as soon as they are added your profile will show the correct icons.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Santosh1176** [comment]
👍

File: public/data/santoshdts.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      {
+        "title": "Started building my profile page",
+        "date": "April 2022",
+        "icon": "HTML5 and CSS3",
+        "color": "blue",
+        "description": "Learning interactive CSS layout"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/santoshdts.json:50]:
```suggestion
}
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by kabir0x23 — APPROVED** [review]
Looks Good 🧨

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 🚀

**Comment by Santosh1176** [comment]
Hi, do I need to change the milestone icons to default available ones?

**Comment by emmalearnscode** [comment]
> Hi, do I need to change the milestone icons to default available ones?

No, there is an automatic default built in. However, you should open an issue asking for them to be added. Once they’re added they’ll automatically show correctly

**Comment by Santosh1176** [comment]
That's great !
👍

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**